params/config.go

```diff
@@ -20,11 +20,13 @@
 
 package params
 
+import "C"
```

Comment:
Why was this pkg added?

Reply:
I don't know why ... It is removed in `193db0b`